environment.yml

```diff
@@ -17,8 +16,8 @@ dependencies:
   - s3fs
   - universal_pathlib <0.2.0
   - boto3
-  - deltalake=0.15.3
+  - deltalake >=0.18
   # - dask-deltatable
   - pip
   - pip:
-    - git+https://github.com/fjetter/dask-deltatable.git@dask_expr
+    - dask-deltatable==0.3.3
```

Comment:
Can remove this now

Reply:
I had to pip install it for things to work

Reply:
Hmm that's strange but 🤷